scripts/circle_deploy.sh

```diff
@@ -10,26 +10,18 @@ git config --global user.name "tadeploy"
 git config --global user.email "devteam@technologyadvice.com"
 
 #
-# build
-#
-gulp docs
-
-#
 # generate changelog
 #
 echo "...generating changelog"
-github_changelog_generator $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
+curl -X POST github-changelog-api.herokuapp.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME \
```

Comment:
I was going to ask if the default machines came with `curl` installed (as opposed to just `wget`, for many vanilla *nix boxes), then I saw the build pass. ^^